uint8 direction
float32 amplitude

uint8 repetition
float32 period_render
float32 period_pause 
